chapters/en/slides/module5/module5_05.md

```diff
@@ -1,3 +1,241 @@
+
+Notes: Now that all the collaborators have access they can clone the repository and create a branch using a meaningful name with the feature they are willing to work on. If some of the collaborators push to `main` without socializing it to the other team members first could create merging problems. To avoid that risk, you can set a [branch protection rule](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/managing-a-branch-protection-rule) in the repository making it require a pull request before merging. We will talk with detail about pull requests later on, but for now you can think about them as specific notifications you will receive when someone is trying to modify the `main` branch. Pull request should be approved before merging in general by other team members.
+
+🙌
+To add a branch protection rule you should go to **Settings > Code and Automation (sidebar) > Branch protection rules** and click in **"Add rule"** in your repository. 
+
+---
+
+# Branch protection rules
+
+
+
+<center>
+
+<img src='/module5/PR-reviews-required-updated.png' width="80%" alt="404 image" />
+
+<center/>
+
+Notes:
+
+🙌
+Under "Branch name pattern", type the branch name or pattern you want to protect. 
+Notice that you can create protect branches different to `main` but we will not cover those cases of use in this tutorial.
+
+---
+
+# Branch protection rules
+
+<center>
+
+<img src='/module5/branches-house-analogy-explained.png' width="80%" alt="404 image" />
+
+<center/>
+
+Notes:
+
+If you think in the analogy of the house, protecting the `main` branch would be similar to avoid creating modifications in the structure of the house meanwhile each member is working in different sections. It is not a good idea to modify the foundation of the house at the same time other team members are working in the walls or windows. If there are changes in the house's foundation there could be problems when combining the parts, including the risk of collapsing the house.
+
+In your repository this means that you should be sure that code in `main`, also called the production branch, is high quality code without bugs or other run problems. Each time you accept merging a contribution of another team member in `main` be sure that the code is doing what is expected and it is well documented. This way you will always have stable and well build code that runs propoperly in your `main` branch. This ensures that you are producing code collaboratively in an organized way.
+
+Using branch protection rules is a team desicion that could avoid mistakes of pushing to the `main` branch without notifying your colleagues. If you made only minor edits the team might let you merge these to `main` locally and then push up the updated `main` branch (or even let you not create a branch in the first place).
+
+⚠️ Remember to share with other team members that you have stablish this dynamic of work in the repository
```

Comment:
```suggestion
⚠️ Remember to get the input from other team members before deciding to implement branch protection, so that everyone in your team is on the same page.
```